### DISCUSSION TITLE
Which is the best setting for the new BambuLab Supertack plate?

### DISCUSSION THREAD

**Discussion by ForestEco** [OrcaSlicer/OrcaSlicer#7846] [General]
To reiterate the title, which is the best setting for the new BambuLab Supertack plate? I have not found anything about it on the forum to my surprise.

**Comment by quartzar** [comment]
Following up - also wondering this. Is it expected to be added as a plate option soon?

**Reply by quartzar** [reply]
Looks like this has been merged already #7664 #7629 
Can be used by grabbing the latest nightly build from [https://github.com/SoftFever/OrcaSlicer/releases](https://github.com/SoftFever/OrcaSlicer/releases)
